packages/juce-blueprint/src/index.tsx

```diff
@@ -0,0 +1,52 @@
+/** Polyfill ES2015 data structures with core-js. */
+import 'core-js/es6/set';
+import 'core-js/es6/map';
+
+import './lib/polyfill';
+
+import BlueprintBackend from './lib/BlueprintBackend';
+import BlueprintRenderer, { BlueprintTracedRenderer } from './lib/BlueprintRenderer';
+
+export { default as NativeMethods } from './lib/NativeMethods';
+export { default as EventBridge } from './lib/EventBridge';
+
+export * from './components/View';
+export * from './components/ScrollView';
+export * from './components/Canvas';
+export * from './components/Text';
+export * from './components/Image';
+export * from './lib/SyntheticEvents';
+
+let __renderStarted = false;
+let __preferredRenderer = BlueprintRenderer;
+
+export default {
+
+  getRootContainer() {
+    return BlueprintBackend.getRootContainer();
+  },
+
+  render(element: any, container: any, callback?: () => void | null | undefined) {
+    console.log('Render started...');
+
+    // Create a root Container if it doesnt exist
+    if (!container._rootContainer) {
+      //TODO: Double check passing false for final param "hydrate correct"
+      container._rootContainer = __preferredRenderer.createContainer(container, false, false);
+    }
+
+    // Update the root Container
+    // TODO: callback in __preferredRenderer.updateContainer is not optional.
+    // @ts-ignore
+    return __preferredRenderer.updateContainer(element, container._rootContainer, null, callback);
+  },
+
+  enableMethodTrace() {
```

Comment:
This whole thing was really just a debug tool for myself. I use it to get details about which HostConfig methods React is calling and when. The HostConfig is now mostly implemented, so we're nearly to a point where we can just remove this. Really just think we'll need this here when upgrading React and the reconciler to support their fiber implementation and to see if we can get hooks for committing a batch of updates.

All of this is future work, just highlighting what this is for and that it should probably go away shortly.